Free Worlds campaign rework (Stage 1) (#7000)
endless-sky/endless-sky: data/human/free worlds start.txt

	landing
	name "Tarazed Diplomacy"
	description "Transport Alondo Gruyere to <planet> to negotiate with Tarazed Corporation."
	source "Hephaestus"
	destination "Wayfarer"
	clearance "Alondo has a brief chat with the spaceport controller and manages to get you permission to land."
		attributes "dirt belt"
	passengers 1
	to offer
		has "FW Syndicate Diplomacy 1: done"
	on fail
		dialog `You have failed an essential Free Worlds mission. If you want to complete the story line, revert to the autosave or another earlier snapshot of the game.`
	
	on offer
		log "Visited the Syndicate and worked out a deal where they promised to stay neutral. They are expecting to profit from selling weapons to both sides in this war. Also discovered that Katya is in the Republic's custody and is accused of terrorist activity."
		log "People" "Katya Reynolds" `Katya was on Clink when the Navy invaded the Kornephoros system, and was taken captive by them and imprisoned as a war criminal.`

> 		log "People" "Samuel Remington" `Samuel Remington is the Syndicate's head of public affairs, and he seemed to help get the Free Worlds a better deal with the Syndicate than they might have otherwise received. He and Alondo seem to have a history together, although it is one that neither will discuss in detail.`

Author: ```suggestion
		log "People" "Samuel Remington" `Samuel Remington is the Syndicate's head of public relations, and he seemed to help get the Free Worlds a better deal with the Syndicate than they might have otherwise received. He and Alondo seem to have a history together, although it is one that neither will discuss in detail.`
```

Reviewer: Did you request this as a reminder, or are you calling for discussion here?

Reviewer 2: It's a mirror to a thing in my PR, and also probably a nudge for me to see that this is where stuff is for my reference.

			`	"Piracy is on the rise in the Dirt Belt," says Alondo. "And I know you have customers there. Such as the Tarazed Corporation. If we can count you among our friends, we could ensure that your convoys reach Tarazed unmolested."`
			`	"And you would no doubt immediately purchase said supplies from Tarazed," says Badenoch.`
			`	A much older man sitting next to him, Akihito, says, "Of course, such weapons would end up being bought at a significant price markup." He is grinning.`
			`	"No," Remington says, surprising some of the other men at the table. "No markups necessary." None of the other men argue with Remington over this.`
			`	You speak for a while longer, but Alondo manages to hash out a deal where the Free Worlds will defend any Syndicate ships they encounter within the Dirt Belt in return for a promise that the Syndicate will not serve as Republic mercenaries and will not prevent you from purchasing their equipment indirectly through Tarazed. In addition, Remington suggested that you might find some "extra goodies" among the Syndicate convoys heading to Tarazed specifically. You thank the Syndicate leaders, and begin the trip back to your ship.`

> 			`	Alondo is silent. As you're about to leave the spaceport for the landing pads, you hear someone shout behind you. "No need to be a stranger, Gruyere. Can't stick around for a drink?" You turn to look at who it is, but Alondo keeps walking. You find Samuel Remington approaching, paying you no mind. "Are you still mad at me, Gruyere?" he asks`

Reviewer 2: Just gonna say, the dynamic through this section had me rolling. Not sure if it was intended but it's so specific I'm pretty sure I could summarise their whole background off this interaction alone.

			label remington
			choice
				`	"He's clearly not interested"`
					goto interested
				`	"You know Alondo?"`
			
			`	"More than most," Remington says with a chuckle. "He was never one to show gratitude for gifts. I hope you enjoy the new toys in those shipments I just secured more than he does."`
				goto alondo
			
			label interested
			`	"As if that's ever stopped me before," Remington says with a chuckle. "He was never one to show gratitude for gifts. I hope you enjoy the new toys in those shipments I just secured more than he does."`
			
			label alondo
			`	"What new toys?" you ask.`
			`	"You'll see," he says, turning back to the spaceport and leaving you.`
			`	You join Alondo on your ship and find him solemnly staring out a window.`
			choice
				`	"It seems like the negotiations went pretty well."`
					goto well
				`	"What was that all about?"`
					goto that
				`	"What do we do about Katya?"`
					goto katya
			
			label that
			`	"Samuel and I have... a history. One I'd rather not discuss," Alondo responds. "Anyway, we must continue with our mission. Time for us to make a visit to Tarazed. Let's get going."`
				accept
			
			label well

> 			`	"They did. Although it feels like it was almost too easy." Alondo pauses for a moment. "Anyway, we must continue with our mission. Time for us to make a visit to Tarazed."`

Author: I dig it.

			`	Hines turns up his nose. "This is undoubtedly Republic territory you're standing on, Soleau, and therefore we have every right to arrest you."`
			`	"I think your men disagree," JJ remarks. All of the Navy personnel have either remained in or returned to formation. "Remember that I myself was a part of the Navy once. A Navy that each and every one of your honored men surely remember, either because they lived it or because they grew up revering it. But a Navy that I left because of those like you. I'm sorry to disappoint you, but if we can't negotiate peace talks then we'll be leaving. I pray that you'll be wise enough to see the error in your ways and let us go." JJ stands up and begins walking off, leaving Hines to sulk in his chair.`
			`	"Your days are numbered, Soleau," Hines shouts, but he remains in his chair and gives no orders to his soldiers.`

> 			`	The Navy allows you to leave unmolested as you enter your ship with JJ. "Launch your ship immediately and return to <planet>," he says. "The only thing that rivals honor among the Navy is duty, and I don't want to be around with Hines starts whipping his men into his shape."`

Reviewer 3: ```suggestion
			`	The Navy allows you to leave unmolested as you enter your ship with JJ. "Launch your ship immediately and return to <planet>," he says. "The only thing that rivals honor among the Navy is duty, and I don't want to be around when Hines starts whipping his men into his shape."`
```

			`	"That is yet to be seen," says Freya, "but right now we're in the red. Do you know how much we're going to need to spend on repairs as a result of this stunt of yours, not only in credits but in man-hours? A massive amount of our fleet is out of commission because of this. If the Navy attacked now we'd be practically defenseless."`
			`	Tomek pauses as he notices you standing at the entryway to the tent. Freya and Alondo also look over to you. "Care to explain yourself?" Freya asks.`
			choice
				`	"I was simply following the orders of a superior. This was Tomek's idea."`
				`	"No."`
					goto no
			`	"The orders of your superior were the orders from the Senate to escort me to Thule for a diplomatic mission," Alondo responds angrily. "Now that I'm late and word of this has spread quickly, the people of Thule are not interested in negotiating. I'm sure the Senate would love to hear your explanation. You know full well that you should not have come here, yet you did. You, Tomek, and every other captain here will be answering to the Senate shortly."`
				goto emergency
			
			label no
			branch live
				random < 99

> 			`	Freya heaves a deep sigh and looks to the side for a moment before reaching to her hip, drawing her sidearm, and shooting you. You do not have an opportunity to react. You must have caught Freya on a particularly bad day. Try again.`

Reviewer 4: I do so love this.

	source "Longjump"
	destination "Deep"
	to offer
		has "FW Pirates: Diplomacy 3: done"
	on fail
		dialog `You have failed an essential Free Worlds mission. If you want to complete the story line, revert to the autosave or another earlier snapshot of the game.`
	
	on offer
		event "fw suppressed Greenrock"
		conversation
			`You arrive back on Longjump to a hero's welcome. People have heard of the battle you fought on Thule and the deal that was forged with the locals there. JJ and Freya greet you too, and congratulate you. But then Freya says, "We need to talk in private."`
			`	You lead them into your ship, where Freya explains, "Soon after you left for Thule, Tomek took off on his own and raised a fleet for an attack on Greenrock. He thought the Senate's plan of making a deal with the pirate worlds was too cowardly, and that we needed a show of strength instead. Surprisingly, his stunt succeeded; Greenrock is now nominally under Free Worlds control."`

> 			`	"This puts us in a rather difficult position," says JJ. "Much of Tomek's fleet didn't survive the assault, and those ships didn't come out of thin air, they were our ships intended for defensive purposes in case the Navy attacks again. Word of this has spread quickly. Thule is already second guessing their deal with us, and if Thule has heard of this then there's no doubt in my mind that the Republic has as well."`

Reviewer 4: ```suggestion
			`	"This puts us in a rather difficult position," says JJ. "Much of Tomek's fleet didn't survive the assault, and those ships didn't come out of thin air; they were intended for defensive purposes in case the Navy attacks again. Word of this has spread quickly. Thule is already second guessing their deal with us, and if Thule has heard of this then there's no doubt in my mind that the Republic has as well."`
```

			`	Hines turns up his nose. "This is undoubtedly Republic territory you're standing on, Soleau, and therefore we have every right to arrest you."`
			`	"I think your men disagree," JJ remarks. All of the Navy personnel have either remained in or returned to formation. "Remember that I myself was a part of the Navy once. A Navy that each and every one of your honored men surely remember, either because they lived it or because they grew up revering it. But a Navy that I left because of those like you. I'm sorry to disappoint you, but if we can't negotiate peace talks then we'll be leaving. I pray that you'll be wise enough to see the error in your ways and let us go." JJ stands up and begins walking off, leaving Hines to sulk in his chair.`
			`	"Your days are numbered, Soleau," Hines shouts, but he remains in his chair and gives no orders to his soldiers.`
			`	The Navy allows you to leave unmolested as you enter your ship with JJ. "Launch your ship immediately and return to <planet>," he says. "The only thing that rivals honor among the Navy is duty, and I don't want to be around when Hines starts whipping his men into his shape."`

> 				launch

Reviewer 4: This whole section is great :)

	
	on enter "Kornephoros"

> 		dialog `Upon entering <system>, you spot Free Worlds ships in combat with the Navy. It looks like the Navy only sent a small force so far, as there are only a few ships remaining and there don't appear to be many destroyed ships, but the Free Worlds captains defending the system would probably appreciate help clearing out the rest before you land.`

Reviewer 4: ```suggestion
		dialog `Upon entering <system>, you spot Free Worlds ships in combat with the Navy. It looks like the Navy only sent a small force so far; there are only a few ships remaining and there don't appear to be many destroyed ships. Regardless, the Free Worlds captains defending the system would probably appreciate help clearing out the rest before you land.`
```

			`	Finally, a messenger comes and hands you a small slip of paper. You open it. The message reads: "We have decided that you will be suspended without pay and stripped of your rank. When we are in need of your services again, we will contact you, and your rank will be reinstated as the Senate or Council sees fit." It looks like for now, you are back to being a simple merchant captain.`

event "fw stripped of service"
	government "Escort"
		swizzle 5

mission "FW Refinery 1"
	autosave
	landing
	name "Smuggler's Den Repairs"
	description "Travel to <planet> and pick up a shipment of equipment from the Syndicate for repairing the refinery on Smuggler's Den."
	source "Longjump"
	destination "Tundra"
	clearance
	to offer
		has "FW Early Warning 5: done"
		has "FW Pirates: Diplomacy 4: done"
	on fail
		dialog `You have failed an essential Free Worlds mission. If you want to complete the story line, revert to the autosave or another earlier snapshot of the game.`
	
	on offer
		conversation

> 			`JJ and Freya meet you soon after you land on <origin>. "Great work in the battle, Captain," Freya says. "Unfortunately, we not need to turn our attention to cleaning up Tomek's mess."`

Reviewer 3: ```suggestion
			`JJ and Freya meet you soon after you land on <origin>. "Great work in the battle, Captain," Freya says. "Unfortunately, we now need to turn our attention to cleaning up Tomek's mess."`
```

			choice
				`	"How are we going to do that?"`
					goto repairs
				`	"So what happens to Tomek now?"`
			`	"Tomek is on his way to <planet> right now," Freya says. "He left as soon as the news of the victory arrived. I imagine we'll know the outcome of his Senate hearing once you've returned from your mission."`
			`	"What mission?" you ask.`
			label repairs
			`	"Well, thanks to you, the Men system is no longer engaged in piracy," says JJ. "And that means the folks on Smuggler's Den are in need of new work. That station used to be a deuterium refinery. With a little work, it can be one, again."`
			`	"A working refinery could bring more income to that system than piracy ever did, and helping to repair it would show the people of Thule that we mean them no harm," says Freya. "Only problem is, the replacement parts are rare, and hard to come by. The Syndicate agreed to sell us the parts, but only if we meet them halfway, on <planet>. That's in the <system> system. Just one jump away from one of the major new Navy bases."`
			choice
				`	"We're going to take a supply convoy through Republic space?"`
					goto convoy
				`	"How do we know this isn't a trap?"`

> 			`	"We don't," says Freya, "by the Syndicate has helped us before, and they are always willing to make deals. I don't think they're ready yet to give up on the chance to sell their wares to both sides in this war."`

Reviewer 3: ```suggestion
			`	"We don't," says Freya, "but the Syndicate has helped us before, and they are always willing to make deals. I don't think they're ready yet to give up on the chance to sell their wares to both sides in this war."`
```

				`	"I trusted that Tomek would know the best strategy."`
				`	"I guess I got too caught up in trying to be a hero."`
					goto hero
			`	"And how did that work out?" says the old man. "Shortly after your attack on Greenrock, the Navy made moves to invade us once again, a plan that was no doubt helped by your actions."`
				goto sentence
			label hero
			`	"Indeed," says the senator. "And I bet it wasn't long after the battle that you realized saving the galaxy isn't that easy."`
			label sentence
			`	"I suppose so," you admit.`
			`	A third senator speaks up, a woman whose name card identifies her as Sara, from Skymoot. "That will be all, Captain <last>. The Senate will confer and decide whether we want your continued services."`

> 			`	You are led back out to the lobby, where you wait for what feels like another hour. During that time, you speak with someone else and learn that Tomek has been removed from the Council and put under house arrest. Although interestingly, despite Tomek disobeying orders, the Senate has decided to continue to occupation of Greenrock, at least for now, their public reasoning being that it would "dishonor those who lost their lives doing what they believed to be right" to leave the planet so quickly.`

Reviewer 4: ```suggestion
			`	You are led back out to the lobby, where you wait for what feels like another hour. During that time, you speak with someone else and learn that Tomek has been removed from the Council and put under house arrest. Although interestingly, despite Tomek disobeying orders, the Senate has decided to continue the occupation of Greenrock, at least for now, because, “it would dishonor those who lost their lives doing what they believed to be right to leave the planet so quickly.”`
```

			`	Finally, a messenger comes and hands you a small slip of paper. You open it. The message reads: "We have decided that you will be suspended without pay and stripped of your rank. When we are in need of your services again, we will contact you, and your rank will be reinstated as the Senate or Council sees fit." It looks like for now, you are back to being a simple merchant captain.`

event "fw stripped of service"
	government "Escort"
		swizzle 5

mission "FW Refinery 1"
	autosave
	landing
	name "Smuggler's Den Repairs"
	description "Travel to <planet> and pick up a shipment of equipment from the Syndicate for repairing the refinery on Smuggler's Den."
	source "Longjump"
	destination "Tundra"
	clearance
	to offer
		has "FW Early Warning 5: done"
		has "FW Pirates: Diplomacy 4: done"
	on fail
		dialog `You have failed an essential Free Worlds mission. If you want to complete the story line, revert to the autosave or another earlier snapshot of the game.`
	
	on offer
		conversation
			`JJ and Freya meet you soon after you land on <origin>. "Great work in the battle, Captain," Freya says. "Unfortunately, we now need to turn our attention to cleaning up Tomek's mess."`

> 			`	"The people of Thule became very weary of our deal with them after Tomek's attack on Greenrock," JJ explains. "They feared that they would be next, and that our negotiations with them was just an attempt to get them to distance themselves from what would be their main line of defense."`

Reviewer 4: ```suggestion
			`	"The people of Thule became very weary of our deal with them after Tomek's attack on Greenrock," JJ explains. "They feared that they would be next, and that our negotiations with them were just an attempt to get them to distance themselves from what would be their main line of defense."`
```

		dialog `You have failed an essential Free Worlds mission. If you want to complete the story line, revert to the autosave or another earlier snapshot of the game.`
	
	on offer
		conversation
			`Your landing site is an abandoned settlement a thousand kilometers from <origin>'s main spaceport. It's an old oil field that ran dry decades ago, and only a few of the landing pads are still usable. The Syndicate employees quickly load the cargo onto your ship, and say, "Now, take off quickly, before the Navy sees us doing business with a ship flying Free Worlds colors."`
				launch
	
	on visit
		dialog phrase "generic cargo on visit"
	on complete
		event "wolf pack reveal" 15
		set "fw first chapter done"
		log "Brought supplies to Smuggler's Den to help upgrade the station into a working deuterium refinery. No further orders from the Free Worlds, so this might be a good time to find ways to earn more money on the side."
		log "People" "Tomek Voigt" `Tomek was removed from the Council and placed under house arrest by the Senate after he disobeyed their orders and captured the pirate world of Greenrock rather than pursue the diplomatic solution that the Senate preferred.`
		conversation

> 			`After unloading the supplies, you send Freya and JJ a message letting them know that you arrived safely with the refinery equipment. Workers on <origin> along with Free Worlds militia members work to throw out the old, rusted hardware and install the new equipment. It will likely take a few days for everything to get up and running again, but the people of this system seem eager to get back to more honest work.`

Reviewer 4: ```suggestion
			`After unloading the supplies, you send Freya and JJ a message letting them know that you arrived safely with the refinery equipment. Workers from <origin> work alongside Free Worlds militia members to throw out the old, rusted hardware and install the new equipment. It will likely take a few days for everything to get up and running again, but the people of this system seem eager to get back to more honest work.`
```
